Description: Correct a typographical error.
Forwarded: yes
Author: Peter Pentchev <pp@storpool.com>
Last-Update: 2019-01-31

--- a/bin/netplan-parser
+++ b/bin/netplan-parser
@@ -117,7 +117,7 @@
 
 def cmd_physical(args):
     """
-    Gather the configuartion of the specified interfaces and all other
+    Gather the configuration of the specified interfaces and all other
     interfaces related to them, e.g. raw devices for VLANs, members for
     bridge and bond interfaces, etc; then show the configuration for
     the physical interfaces only.
